editor/hooks/saved-post-url/index.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies.
+ */
+import { get } from 'lodash';
+
+/**
+ * WordPress dependencies.
+ */
+import { addAction } from '@wordpress/hooks';
+
+/**
+ * Internal dependencies.
+ */
+import { getPostEditUrl } from '../../utils/url';
+
+addAction(
```

Comment:
I'm not sure if this hooks should be inside `editor` module or rather inside `edit-post` because it is very WordPress specific and also might not work with CPTs or Customization work planned ahead. @youknowriad what's your opinion on this?

Technically it's only refactoring, but since we are touching it I wanted to make sure we put it in a right place as it won't be much of the work :)

Reply:
Related #6287

Maybe the approach here is not the usage of a hook but more moving the saving/fetching post behavior outside of the editor into edit-post making the editor a generic editor module.